### DISCUSSION TITLE
Burner max power vs. burner selected max power

### DISCUSSION THREAD

**Discussion by pautri** [emsesp/EMS-ESP32#919] [Q&A] [answered]
Hi all, I was wondering what the difference is between the "burner max power" and "burner selected max power" parameters. If I change the max power on the boiler itself, both seem to be changed to the same value. I thought that one of them was perhaps controlled by the boiler itself, but both are user adjustable in the EMS-ESP interface, so that would argue against that interpretation. Any ideas?

**Comment by pautri** [comment, marked as answer]
I've been monitoring both values for some days now in Home Assistant and "burner selected max power" is indeed modulated (to a value below "burner max power") by the boiler itself at times, whereas "burner max power" remains on the value I entered. Still a bit odd that "burner selected max power" is user adjustable as well.

**Reply by guardianbs** [reply]
See Discussion #1023 for my observations.
